pybleau/plotly_api/plotly_scatter.py

```diff
@@ -306,12 +307,19 @@ def plotly_scatter(x="", y="", data=None, z=None, hue=None,
         # Build the Scatter object --------------------------------------------
 
         marker_line = {"width": 0.5, "color": 'rgba(217, 217, 217, 0.14)'}
+
+        # Allow x to be empty:
+        if x:
+            x_data = subdf[x]
```

Comment:
Should we also check that `x` is a column in `subdf` before performing this operation? Also wondering why we didn't go the same route with `x` that we did with `z` (where we check `if z is None:` below.